sormas-ui/src/main/java/de/symeda/sormas/ui/MainScreen.java

```diff
@@ -225,6 +236,50 @@ public View getView(String viewName) {
 
 		navigator.addViewChangeListener(viewChangeListener);
 
+		// Add GDPR window
+		// possible to desactivate it with check
+		UserDto user = FacadeProvider.getUserFacade().getCurrentUser();
+		if (user.isGdpr()) {
+			Window subWindowGdpR = new Window(I18nProperties.getCaption(Captions.User_gdpr));
```

Comment:
Use `I18nProperties.getPrefixCaption` instead and directly reference the static String in UserDto